ROADMAP.md

```diff
@@ -132,22 +132,49 @@ Any dates listed below and the specific issues that will ship in a given milesto
 
 * Bug fixes and test automation
 
-## Under consideration for v0.6.0
+## [v0.6.0 Aggregated Stack Roles, GKECluster to v1beta1, test automation](https://github.com/crossplaneio/crossplane/releases/tag/v0.6.0)
+* The Stack Manager supports more granular management of permissions for cluster (environment) and namespace (workspace) scoped stacks.
+  * Default admin, editor, and viewer roles automatically updated as Stacks are installed/uninstalled.
+  * Admins can create role bindings to these roles, to simplify granting user permissions.
+  * Details in the [design doc](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-stacks-security-isolation.md).
+* GKE cluster support has moved to `v1beta1` with node pool support.
+  * The `v1alpha3` GKE cluster support has been left intact and can run side by side with v1beta1 
+* Integration test framework in the crossplane-runtime, reducing the burden to provide integration test coverage across all projects and prevent regressions.
+* Helm 2 and 3 compatibility, Crossplane and all of its CRDs are supported to be installed by both Helm2 and Helm3
+* Design and architecture documents:
+  * ["Easy config stacks"](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-resource-packs.md)
+  * [Consuming any Kubernetes cluster for workload scheduling](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-consuming-k8s-clusters.md)
+  * [User experience for template stacks](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-template-stacks-experience.md).
+* [Bug fixes and other closed issues](https://github.com/crossplaneio/crossplane/milestone/6?closed=1)
+
+## v0.7.0
+  * KubernetesTarget kind for scheduling KubernetesApplications [#859](https://github.com/crossplaneio/crossplane/issues/859)
+  * Versioning and upgrade support [#879](https://github.com/crossplaneio/crossplane/issues/879)
```

Comment:
Versioning and upgrade support appears in both 0.7.0 and the roadmap.  It's a broad concept.  
I like it on the roadmap, but I don't know what if any pieces of this work will land by 0.7.0.

Reply:
Is this suggesting that a one-pager will be in place?

https://github.com/crossplaneio/crossplane/pull/937 is stale but can certainly be revisited for 0.7.0, possibly making it in.  I expect I will have a broader one pager around versioning and upgrading available, but I don't know that it will land by 0.7.0 and this could affect #937.

Reply:
@displague it's just the #435 design that's in v0.7, under the umbrella #879